back/engines/commercial/nlp/app/services/nlp/api.rb

```diff
@@ -142,6 +142,11 @@ def text_network_analysis(tenant_id, input_identifiers, max_nodes: nil, min_degr
     # @return [Array]
     def geotag(tenant_id, text, locale, options = {})
       body = options.merge(text: text, locale: locale)
+      
+      if body[:text].nil?
```

Comment:
We added this check first, because `body[:text]` was sometimes `Nil` which the NLP couldn't deal with. But in retrospect this might have been because of the issue above. So not sure if this is needed anymore, but having an extra sanity check doesn't hurt I guess

Reply:
Actually, if the check is not needed, I'd remove it. (After quickly looking in the code, seems it does not.) Otherwise, I'd rather check this before calling `NLP::Api#geotag`.